src/main/resources/standardTestCases/monitoring_log/monitoring_log.options

```diff
@@ -0,0 +1,3 @@
+{
+    "monitoring_script": "gs://cloud-cromwell-dev/some/faux_script.sh"
```

Comment:
I'm actually not totally sure what's happening here.  Is this "faux" script actually producing a file called "monitoring.log"?  And it's faux because it's not actually monitoring but just making a dummy file?